Generator/Parser/JsonSchemaParser.php

```diff
@@ -0,0 +1,102 @@
+<?php
+
+namespace WsSys\DtoGeneratorBundle\Generator\Parser;
+
+use WsSys\DtoGeneratorBundle\Generator\Parser;
+use WsSys\DtoGeneratorBundle\Generator\Parser\Interfaces\ParserInterface;
+
+/**
+ * Reads Json and it's elements
+ */
+class JsonSchemaParser implements ParserInterface
+{
+    
+    /**
+     * @var RecursiveIteratorIterator 
+     */
+    private $start;
+    
+    /**
+     * Reads the Json from given source
+     * 
+     * @param string $source
+     */
+    public function parse($source)
+    {
+        $json = file_get_contents($source);
+        $jsonIterator = new \RecursiveIteratorIterator(
+                new \RecursiveArrayIterator(json_decode($json, TRUE)), 
+                \RecursiveIteratorIterator::SELF_FIRST
+            );
+
+        $this->start = $jsonIterator;
```

Comment:
It might be a good idea to have it as fluent interface to chain better with the other method of the class.